.github/workflows/java-publish-sonatype.yml

```diff
@@ -1,13 +1,12 @@
 name: Build and release Java distributions to sonatype.
 
 on:
-  workflow_dispatch:
   push:
     branches:
-    - master
-    - version-*
+      - master
```

Comment:
shouldn't dev-version-* also be here?

Reply:
No, because we should add a separate workflow for that